src/transformers/models/visual_bert/modeling_visual_bert.py

```diff
@@ -1307,10 +1060,35 @@ def forward(
         labels=None,
     ):
         r"""
-        labels (:obj:`torch.LongTensor` of shape :obj:`(batch_size,)`, `optional`):
-            Labels for computing the multiple choice classification loss. Indices should be in ``[0, ...,
-            num_choices-1]`` where :obj:`num_choices` is the size of the second dimension of the input tensors. (See
-            :obj:`input_ids` above)
+            labels (:obj:`torch.LongTensor` of shape :obj:`(batch_size,)`, `optional`):
+                Labels for computing the multiple choice classification loss. Indices should be in ``[0, ...,
+                num_choices-1]`` where :obj:`num_choices` is the size of the second dimension of the input tensors.
+                (See :obj:`input_ids` above)
+
+        Example::
+
+            >>> from transformers import BertTokenizer, VisualBertForMultipleChoice
+            >>> import torch
+
+            >>> tokenizer = BertTokenizer.from_pretrained('bert-base-uncased')
+            >>> model = VisualBertForMultipleChoice.from_pretrained('gchhablani/visualbert-vcr')
+
+            >>> prompt = "In Italy, pizza served in formal settings, such as at a restaurant, is presented unsliced."
+            >>> choice0 = "It is eaten with a fork and a knife."
+            >>> choice1 = "It is eaten while held in the hand."
+
+            >>> visual_embeds = get_visual_embeddings(image)
+            >>> visual_embeds = visual_embeds.expand(1, 2, *visual_embeds.shape) # (batch_size, num_choices, visual_seq_length, visual_embedding_dim)
```

Comment:
Put the comment above to avoid a line too long

Reply:
Will fix.